### PR TITLE
[FIX]: Minor Documentation fix for Zsh Configuration

### PR DESCRIPTION
**Pull request by N0vice17**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -404,7 +404,7 @@ Edit the value of `ZSH_THEME` to "`xiong-chiamiov`" like this:
 
 `ZSH_THEME="xiong-chiamiov"`
```
Comment (Arkapravo-Ghosh) [inline README.md:406]: How about this as well?
```suggestion

Save (Ctrl + S) and Exit (Ctrl + X)

```

**Review by N0vice17 — COMMENTED** [review]
[FIX]:Fixing Commands int the ZSH portion

**Comment by Arkapravo-Ghosh** [comment]
> [FIX]:Fixing Commands int the ZSH portion

That should've been the title, not a comment. I'll edit that for you.

File: README.md (hunk 1 above)
Comment (Arkapravo-Ghosh) [reply README.md:406]: @N0vice17 Review this.

**Review by Arkapravo-Ghosh — APPROVED** [review]

**Review by Arkapravo-Ghosh — APPROVED** [review]

**Pull request merged**